src/reconstruction/refinement/updateMetNames.m

```diff
@@ -0,0 +1,30 @@
+function [updatedModel] = updateMetNames(originalModel, modelToUpdate)
+% Updates model.metNames in a new model, using metNames of an original
+% model.
+%
+% USAGE:
+%
+%    [updatedModel] = updateMetNames(originalModel, modelToUpdate)
+%
+% INPUTS:
+%    originalModel:      COBRA model structure with correct model.metNames
+%    modelToUpdate:      COBRA model structure that needs to have its
+%                        model.metNames updated 
+%
+% OUTPUT:
+%    updatedModel:       COBRA model structure with corrected metNames
+%
+% .. Authors:
+%       - written by Diana El Assal 27/06/2017
+
+indMets = findMetIDs(originalModel, modelToUpdate.mets);
```

Comment:
You mean metNames is wrong? 
I would suggest an option to "wipeExisting", as I would assume that the "normal" situation is that someone wants to take information from another model without removing the current information if there is any.